Modules/BenchPress.Azure/Tests/Public/CustomAssertions/BeSuccessful.Tests.ps1

```diff
@@ -0,0 +1,20 @@
+﻿using module ./../../../Classes/ConfirmResult.psm1
+
+BeforeAll {
+  . $PSScriptRoot/../../../Public/CustomAssertions/BeSuccessful.ps1
+}
+
+Describe "ShouldBeSuccessful" {
+  Context "unit tests" -Tag "Unit" {
+    It "Should be deployed with a ConfirmResult object" {
+      $mockResource = [PSCustomObject]@{
+        Name = 'mockValue'
+      }
+      [ConfirmResult]::new($mockResource, $null) | Should -BeSuccessful
+    }
+  }
+
+  It "Should fail if ConfirmResult is null" {
```

Comment:
can you add a test case where the confirm result object has success == false

Reply:
good catch